config/i18n-tasks.yml

```diff
@@ -82,18 +82,20 @@ search:
 #   api_key: "AbC-dEf5"
 
 ## Do not consider these keys missing:
-# ignore_missing:
+ignore_missing:
+  - 'zxcvbn.*'
```

Comment:
We're explicitly referencing all these keys inside the `js_i18n.erb` file, so we don't need this ignore, right?